addon/templates/components/inline-edit-field.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if isEditing}}
+  <form class="editable is-editing"
+    {{action "stopEditing" model on="submit"}}>
+
+    {{#if (hasBlock "inverse")}}
+      {{yield to="inverse"}}
+    {{else}}
+      {{input value=value}}
+    {{/if}}
+
+    <div class="save-and-cancel">
```

Comment:
Should `yield` be all or nothing? I assume if we want to be able to customize the input we'd also want to be able to customize the submit buttons?